runtime/src/lib.rs

```diff
@@ -723,6 +726,50 @@ impl pallet_tx_pause::Config for Runtime {
 	type WeightInfo = pallet_tx_pause::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+	pub const ProposalBond: Permill = Permill::from_percent(5);
+	pub const ProposalBondMinimum: Balance = 0;
+	pub const ProposalBondMaximum: Option<Balance> = Option::None;
+	pub const SpendPeriod: BlockNumber = 24 * DAYS;
```

Comment:
hmm, actually we might need to create our own treasury pallet which doesn't have these kinds of concepts, since the chain won't necessarily be producing a block every 6 seconds

Reply:
For now, we should set this very low, like 100 blocks maybe

Reply:
Actually, just remembering, we need to make our own pallet which is actually connected to the relay chain block number

Reply:
Ideally feature should be added to treasury

Maybe `SpendPeriod` type should be able to take either a block number or a relay block number. I will look into it